packages/react-recommendations/src/__tests__/RelatedProducts.tsx

```diff
@@ -0,0 +1,136 @@
+import { render, waitFor } from '@testing-library/react';
```

Comment:
The file's name should be `RelatedProducts.test.tsx`.

Reply:
Fixed in https://github.com/algolia/ui-components/pull/26/commits/ece3ce07a79935145c7bfef4488f6525aaa4355a